src/metadata_service/fam_metadata_service_direct.cpp

```diff
@@ -37,14 +37,19 @@
 #include <boost/atomic.hpp>
 
 #include <atomic>
+#include <bits/stdc++.h>
 #include <chrono>
 #include <iomanip>
 #include <sstream>
 #include <string.h>
 #include <unistd.h>
 
+#include "bitmap-manager/bitmap.h"
+#include "common/fam_internal.h"
 #include "common/fam_memserver_profile.h"
 
+#define SIZE_PER_MEMSERVER 4096
```

Comment:
Can you increase this number. Also we will add this to configuration file. We can take that up in next PR if not in this.